src/extractor/pdportal/posting/status.ts

```diff
@@ -0,0 +1,76 @@
+export type JobPostingStatus = "APPROVED" | "EXPIRED" | "UNKNOWN";
+
+export type InternalStatus =
+  | "FILLED"
+  | "PARTIALLY-FILLED"
+  | "UNFILLED"
+  | "EMP-RANKINGS-FINALIZED"
+  | "INTERVIEW-COMPLETE"
+  | "INTERVIEW-SELECTIONS-COMPLETE"
+  | "EXPIRED-APPLICATIONS-AVAILABLE"
+  | "OPEN-FOR-APPLICATIONS"
+  | "CANCEL"
+  | "UNKNOWN";
+
+export type ApplicationStatus =
+  | "APPLIED"
+  | "SHORTLISTED"
+  | "NOT-INTERESTED"
+  | "EXPIRED"
+  | "AVAILABLE"
+  | "UNKNOWN";
+
+export type InterviewStatus =
+  | "APPLIED"
+  | "NOT-SELECTED"
+  | "SELECTED-FOR-INTERVIEW"
+  | "EMPLOYED"
+  | "NONE"
+  | "UNKNOWN";
```

Comment:
Are these all the same on pdportal?

Reply:
`Internal status`
Most of them look familiar, but I don't think 54 applications is enough to judge which ones I haven't encountered yet so I left them all there

`Application Status`
yea these are the same

`InterviewStatus`
these are also the same